src/BodyProgramming/Badr.java

```diff
@@ -31,5 +31,7 @@ public static void main(String[] args) {
         System.out.println(p);
         double q = 1.56642538343;
         System.out.println(q);
+        int a = 7;
+        int b = 8;
```

Comment:
why did you add 2 variables not only one ?

Reply:
Done, I removed variable